Makefile

```diff
@@ -193,73 +165,13 @@ BUNDLE_DEFAULT_CHANNEL := --default-channel=$(DEFAULT_CHANNEL)
 endif
 BUNDLE_METADATA_OPTS ?= $(BUNDLE_CHANNELS) $(BUNDLE_DEFAULT_CHANNEL)
 
-# IMAGE_TAG_BASE defines the docker.io namespace and part of the image name for remote images.
-# This variable is used to construct full image tags for bundle and catalog images.
-#
-# For example, running 'make bundle-build bundle-push catalog-build catalog-push' will build and push both
-# integreatly.org/grafana-operator-experimental-bundle:$VERSION and integreatly.org/grafana-operator-experimental-catalog:$VERSION.
-IMAGE_TAG_BASE ?= integreatly.org/grafana-operator-experimental
-
-# BUNDLE_IMG defines the image:tag used for the bundle.
-# You can use it as an arg. (E.g make bundle-build BUNDLE_IMG=<some-registry>/<project-name-bundle>:<tag>)
-BUNDLE_IMG ?= $(REGISTRY)/$(REPO)/grafana-operator-bundle:v$(VERSION)
-
 .PHONY: bundle
 bundle: manifests kustomize ## Generate bundle manifests and metadata, then validate generated files.
 	operator-sdk generate kustomize manifests -q
 	cd config/manager && $(KUSTOMIZE) edit set image controller=$(IMG)
 	$(KUSTOMIZE) build config/manifests | operator-sdk generate bundle -q --overwrite --version $(VERSION) $(BUNDLE_METADATA_OPTS)
 	operator-sdk bundle validate ./bundle
 
-.PHONY: bundle-build
-bundle-build: ## Build the bundle image.
-	docker build -f bundle.Dockerfile -t $(BUNDLE_IMG) .
-
-.PHONY: bundle-push
-bundle-push: ## Push the bundle image.
-	$(MAKE) docker-push IMG=$(BUNDLE_IMG)
-
-.PHONY: opm
-OPM = ./bin/opm
```

Comment:
Don't know what a bunch of this is, to my knowledge we only need the `make bundle` command to generate all the files needed by OLM. Is any of these commands something that you use in your daily lifes @pb82 @HubertStefanski ?

Reply:
opm is a CLI to build catalog images. I wouldn't say I use it daily, but we will need it for the OLM releases.

Reply:
But I just use `make bundle` to generate the code that I then push up to the central repositories, but I might have been doing it wrong all this time ^^.

But if you use it I will of course not remove it.

Reply:
You don't need it for the bundle, only for the catalog image. Which you don't usually build yourself if you release to operatorhub.io, etc. It is still useful though for testing to be able to build our own catalog images.